lib/formtastic/helpers/enum.rb

```diff
@@ -3,10 +3,8 @@ module Helpers
     # @private
     module Enum
       # Returns the enum (if defined) for the given method
-      def enum_for(method) # @private
-        if @object.respond_to?(:defined_enums)
-          @object.defined_enums[method.to_s]
-        end
+      def enum_for(method) # @private        
+        @object.defined_enums[method.to_s] if @object.respond_to?(:defined_enums)
```

Comment:
@ryanfox1985 would be good to agree on a code style before changing it. I really don't find this more readable.